### PR TITLE
Hook up redux store for notebook on next

### PR DESCRIPTION
**Pull request by lgeiger**
All reducers related to modifying the document *should* work once @rgbkrk's dropdown component is merged.

### CODE REVIEW

File: applications/notebook-on-next/pages/edit.js [hunk 1]
```diff
@@ -25,56 +25,38 @@ async function fetchFromGist(gistId) {
...
-  cellOrder: List<any>,
-  cellMap: Map<string, any>,
-  language: string
-};
-
-class Edit extends React.Component<Props, Props> {
-  static async getInitialProps({ query, res }) {
-    const notebook = await fetchFromGist(query.gistid);
-    return { notebook };
+export default class Simple extends React.Component<*> {
```
Comment (betatim) [inline applications/notebook-on-next/pages/edit.js]: Simple -> Edit?

Comment (lgeiger) [reply applications/notebook-on-next/pages/edit.js]: 👍

File: applications/notebook-on-next/pages/edit.js [hunk 2]
```diff
@@ -25,56 +25,38 @@ async function fetchFromGist(gistId) {
...
-      )
-    );
-    const cellOrder = notebook.get("cellOrder");
-    const cellMap = notebook.get("cellMap");
-    this.state = { cellOrder, language, cellMap };
+  componentWillMount() {
+    if (this.props.serverNotebook) {
+      store.dispatch({
+        type: "SET_NOTEBOOK",
+        notebook: this.props.serverNotebook
```
Comment (betatim) [inline applications/notebook-on-next/pages/edit.js]: do you ever get here with `this.props.serverNotebook` not being true?

**Comment by betatim** [comment]
One nitpick, otherwise LGTM

File: applications/notebook-on-next/pages/edit.js (hunk 2 above)
Comment (lgeiger) [reply applications/notebook-on-next/pages/edit.js]: ~~I think the error came up when github didn't return a notebook. So I added the checks here.~~
Whoops, that's unnecessary

File: applications/notebook-on-next/pages/_document.js [hunk 3]
```diff
@@ -12,6 +12,9 @@ export default class MyDocument extends Document {
     return (
       <html lang="en-US">
         <Head>
+          {/* TODO: Once we're fully migrated to our next gen components we can remove the vendored style sheets */}
+          <link href="/static/normalize.css" rel="stylesheet" type="text/css" />
+          <link href="/static/main.css" rel="stylesheet" type="text/css" />
```
Comment (rgbkrk) [inline applications/notebook-on-next/pages/_document.js]: You should be able to put literally these into a `<style>` block. I also removed pretty much all the normalize too.

Comment (lgeiger) [reply applications/notebook-on-next/pages/_document.js]: :+1: let's fix this

**Pull request merged**